lib/sassone/builder.ex

```diff
@@ -83,6 +83,24 @@ defprotocol Sassone.Builder do
   def root_element(t)
 end
 
+defmodule Sassone.Utils do
```

Comment:
No need for a separate module. Just put this function in the macro generated code.